modules/renter/hostdb/hostweight.go

```diff
@@ -186,7 +186,7 @@ func storageRemainingAdjustments(entry modules.HostDBEntry) float64 {
 // version reported by the host.
 func versionAdjustments(entry modules.HostDBEntry) float64 {
 	base := float64(1)
-	if build.VersionCmp(entry.Version, "1.1.2") < 0 {
+	if build.VersionCmp(entry.Version, "1.1.2") <= 0 {
```

Comment:
ok. The reason this wasn't caught before was that I usually push version bump commits directly to master. If I had made a PR, the build would have failed. We can fix this up when it's time to make the next release.